lib/graphics.py

```diff
@@ -2069,6 +2082,9 @@ def __check_mouse(self, x, y):
 
     """ mouse events """
     def __on_mouse_move(self, scene, event):
+        if self.scale:
```

Comment:
drop the ```if``` here - the "if self.scale" happens also  in get_aspect and this func right now fails if scale is not set (like dragging in convex_hull.py )